presentation/index.html

```diff
@@ -64,6 +64,10 @@
 ### On the roadmap
 
 * Fail-over <!-- .element: class="fragment" data-fragment-index="1" -->
+* Move cfg config files into backend import <!-- .element: class="fragment" data-fragment-index="2" -->
```

Comment:
I would have written: Drop configuration with flat files and load while configuration from the backend (enhance loading time)